sydent/http/servlets/threepidunbindservlet.py

```diff
@@ -31,19 +33,26 @@
     SessionNotValidatedException,
 )
 
+if TYPE_CHECKING:
+    from sydent.sydent import Sydent
+
 logger = logging.getLogger(__name__)
 
 
 class ThreePidUnbindServlet(Resource):
-    def __init__(self, sydent):
+    def __init__(self, sydent: "Sydent") -> None:
         self.sydent = sydent
 
-    def render_POST(self, request):
-        self._async_render_POST(request)
+    def render_POST(
+        self, request: Request
+    ) -> Any:  # from the twisted docs: @type NOT_DONE_YET:
```

Comment:
`NOT_DONE_YET` is an `int`.

```suggestion
    ) -> int:  # from the twisted docs: @type NOT_DONE_YET:
```

I don't know if it worth including the extra comment?

You might also be able to do `Literal[server.NOT_DONE_YET]` but I have no idea if that will work.